CODEOWNERS

```diff
@@ -0,0 +1,6 @@
+# global owners as fallback for anything not matched below
+* @KlausHaeuptleSAP @HrFlorianHoffmann @xtough
```

Comment:
@HrFlorianHoffmann Should we ask Rodrigo and Michel if they could also take over a Owner role?